sdk_v2/kubeflow/training/constants/constants.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2024 The Kubeflow Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+# How long to wait in seconds for requests to the Kubernetes API Server.
+DEFAULT_TIMEOUT = 120
+
+# Common constants.
+GROUP = "kubeflow.org"
+VERSION = "v2alpha1"
+API_VERSION = f"{GROUP}/{VERSION}"
+
+# The default Kubernetes namespace.
+DEFAULT_NAMESPACE = "default"
+
+
+# The Kind name for the ClusterTrainingRuntime.
+CLUSTER_TRAINING_RUNTIME_KIND = "ClusterTrainingRuntime"
+
+# The plural for the ClusterTrainingRuntime.
+CLUSTER_TRAINING_RUNTIME_PLURAL = "clustertrainingruntimes"
+
+
+# The label key to identify training phase where TrainingRuntime should be used.
+# For example, runtime for the pre-training or post-training.
+PHASE_KEY = "training.kubeflow.org/phase"
+
+# The value indicates that runtime can be used for the model pre-training.
+PHASE_PRE_TRAINING = "pre-training"
+
+# The value indicates that runtime can be used for the model pre-training.
+PHASE_PRE_TRAINING = "post-training"
```

Comment:
it should be PHASE_POST_TRAINING I guess

Reply:
Great catch!

Reply:
nit: please change the comment as well 
`The value indicates that runtime can be used for the model pre-training.`
Thanks !